src/generators/clientFileGenerator.ts

```diff
@@ -518,6 +520,38 @@ function getTrack1DefaultContent(
   };
   super(optionsWithDefaults);
   `;
+
+  if (azureArm && !allowInsecureConnection) {
+    return (
+      defaultContent +
+      `
+      if (options?.pipeline && options.pipeline.getOrderedPolicies().length > 0) {
+        const pipelinePolicies: coreRestPipeline.PipelinePolicy[] = options.pipeline.getOrderedPolicies();
+        const bearerTokenAuthenticationPolicyFound = pipelinePolicies.some(
+          pipelinePolicy =>
+            pipelinePolicy.name ===
+            coreRestPipeline.bearerTokenAuthenticationPolicyName
+        );
+        if (!bearerTokenAuthenticationPolicyFound) {
+          this.pipeline.removePolicy({
+            name: coreRestPipeline.bearerTokenAuthenticationPolicyName
+          });
+          this.pipeline.addPolicy(
```

Comment:
@sarangan12 Could you confirm that this default policy will only add if 1) users defined their own pipelines and 2) the defined pipelines didn't contain bearerTokenAuthenticationPolicyName?

What about the user didn't define any options.pipeline? Do we still need to add this as default?

Reply:
@MaryGao The rule is:

1. The code changes affects only the Management Plane SDKs. 
2. For the Management Plane SDKs, if the SDK user does NOT provide any bearerTokenAuthenticationPolicy in options.pipeline, then the new CAE Support Policy will be added. 
3. For the Management Plane SDKs, if the SDK user provides a bearerTokenAuthenticationPolicy, then it will be used and the new CAE Support Policy will not be added. i.e. whatever User provides take precedance.

Reply:
@sarangan12 for the second rule and combined with your code, does that mean if the option.pipeline is empty, we also need to add the CAE policy:
``` typescript
if (!options.pipeline || options.pipeline.getOrderedPolicies().length == 0) {
     this.pipeline.addPolicy(...);
}

```